src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -190,14 +192,37 @@ const getProjectsForSpecies = async (speciesId: number, organizationId: number):
   const apiSearchResults = (await SearchService.search(searchParams)) as SpeciesProjectsSearchResult[];
   // it will always be one result since we are searching by species id
   const firstApiResult = apiSearchResults[0];
```

Comment:
I just noticed that this has the possibility of returning `undefined`, in which case this would fail during runtime. `SearchService.search` returns `SearchResponseElement[] | undefined`.